### ISSUE TITLE
0.32.x update won't compile due to bug in glutin

### ISSUE THREAD

**Issue by john01dav** [glium/glium#2020]
When I updated my project to use glium 0.32.x, it did not compile. I got the following error:
```
error[E0425]: cannot find value `BUFFER_AGE_EXT` in module `ffi::egl`
   --> <removed>/glutin-0.29.0/src/api/egl/mod.rs:584:27
    |
584 |                 ffi::egl::BUFFER_AGE_EXT as i32,
    |                           ^^^^^^^^^^^^^^ not found in `ffi::egl`

error[E0425]: cannot find value `BACK_BUFFER_AGE_EXT` in module `ffi::glx_extra`
   --> <removed>/glutin-0.29.0/src/api/glx/mod.rs:195:33
    |
195 |                 ffi::glx_extra::BACK_BUFFER_AGE_EXT as i32,
    |                                 ^^^^^^^^^^^^^^^^^^^ not found in `ffi::glx_extra`
```

**Comment by est31** [comment]
Yeah I've encountered it myself. It's due to an outdated dependency (don't know which but maybe gl-generator?). A `cargo update` solved it for me. Closing as I would say it's a bug in glutin. They probably have to add put the proper version number in the Cargo.toml requirement. I encourage filing a bug to them.

**Comment by est31** [comment]
This has been fixed by glutin 0.29.1: 30851179a13dbc2bba67cfc78ac49a3087345e83
